docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -37,10 +37,21 @@ per-query basis using the ``writer_min_size`` session property.
 ``use-preferred-write-partitioning``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * **Type:** ``boolean``
-* **Default value:** ``false``
+* **Default value:** ``true``
 
 Enable preferred write partitioning. When set to ``true`` each partition is
 written by a separate writer. As a result, for some connectors such as the
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
 nodes for parallel processing.
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of written partitions that is required to use connector
```

Comment:
`that is` -> `that are`

Reply:
I don't think so. The number is required, not the partitions so it should be singular

Reply:
The number is always there (even if it's 0), so condition "number is required" is always satisfied. However, `N` partitions are required o use preferred partitioning.

@mosabua how this should look like?

Reply:
that is

is correct.